src/components/header/__test__/Header.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { AppContext } from '../../../stores/SearchContext';
+import Header from '../Header';
+
+describe('SearchBar component', () => {
+  const props = {
+    handleStartSearch: () => {},
+    handleStopSearch: () => {},
+  };
+  const apiData = {
+    results: [],
+    total: 0,
+  };
+  const context = {
+    localStorageValue: 'morty',
+    apiData: apiData,
+    setFetchData: () => {},
+    setLocalStorageValue: () => {},
+  };
+  test('renders the Header', async () => {
+    await act(async () => {
+      render(
+        <BrowserRouter>
+          <AppContext.Provider value={context}>
+            <Routes>
+              <Route
+                path="/*"
+                element={<Header {...props} />}
+              />
+            </Routes>
+          </AppContext.Provider>
+        </BrowserRouter>
+      );
+    });
+    expect(true).toBeTruthy();
```

Comment:
Unfortunately this assertion checks nothing =))
Here  you can apply snapshot testing:
```
let tree;
tree = renderSetup(props);
expect(tree).toMatchSnapshot();
```

Reply:
will be fixed in the next PR